dropbox_to_local.py

```diff
@@ -0,0 +1,22 @@
+import dropbox
+import os
+
+dbx = dropbox.Dropbox('OAUTH_API_CREDENTIALS_HERE')
```

Comment:
best practice is to have the API auth key separated in another file and put that file in `.gitignore`

Reply:
I wanted this to be a placeholder for the end-users' own OAuth credentials (using their Dropbox account). If this is not what we wanted, please feel free to let me know! :)